include/bluetooth/hci.h

```diff
@@ -1825,6 +1825,35 @@ typedef bool bt_hci_vnd_evt_cb_t(struct net_buf_simple *buf);
   */
 int bt_hci_register_vnd_evt_cb(bt_hci_vnd_evt_cb_t cb);
 
+
+/** Allocate an HCI event buffer.
+ *
+ * This function allocates a new buffer for a HCI event. It is given the
```

Comment:
text: "an HCI".

Reply:
Actually it should a HCI, since we do pronounce H and even if that was not the case it followed by a consonant not a vogal: https://editingandwritingservices.com/a-or-an-before-words-beginning-with-h/

Reply:
When it comes to abbreviations it is the pronunciation that controls it, for example an FBI agent. I think "an HCI event" ("an aitch si ai event") is correct.

Reply:
I guess that refers to the second term then, an event/an agent, but it wouldn't work for HCI command since that is a command not an command.